db/helpers.js

```diff
@@ -1,44 +1,17 @@
-module.exports.constructGetQuery = (table, id) => {
-  // if id is provided, return the row with the specified id
-  if (id) {
-    return `SELECT * FROM ${table} WHERE id = ${id}`;
-  }
-  // otherwise, return everything from table
-  return `SELECT * FROM ${table}`;
-};
-
-module.exports.constructInsertQuery = (table, data) => {
-  const keys = Object.keys(data);
-  // wrap values in single quotes as part of SQL's syntax
-  const vals = keys.map(key => `'${data[key]}'`);
+module.exports.constructGetOneQuery = table => `SELECT * FROM ${table} WHERE id = $1`;
 
-  const queryKeys = keys.join(', ');
-  const queryVals = vals.join(', ');
+module.exports.constructGetAllQuery = table => `SELECT * FROM ${table}`;
 
-  return `INSERT INTO ${table} (${queryKeys}) VALUES (${queryVals}) RETURNING id`;
+module.exports.constructInsertQuery = (table, fields) => {
+  const queryVals = fields.map((field, index) => `$${index + 1}`).join(', ');
+  return `INSERT INTO ${table} (${fields.join(', ')}) VALUES (${queryVals}) RETURNING id`;
```

Comment:
the `pg` node module prevents SQL injections for you if you use their template strategy. [See here](https://stackoverflow.com/questions/29436565/how-to-properly-escape-raw-sql-query-plainto-tsquery-in-postgres-node) for more details.